content/en/blog/_posts/2024-08-22-cpumanager-static-policy-distributed-cpu-across-cores/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+title: 'Kubernetes v1.31: New Kubernetes CPUManager Static Policy: Distribute CPUs Across Cores'
+date: 2024-08-22
+slug: cpumanager-static-policy-distributed-cpu-across-cores
+author: >
+  [Jiaxin Shan](https://github.com/Jeffwan) (Bytedance)
+---
+
+In Kubernetes v1.31, we are excited to introduce a significant enhancement to CPU management capabilities: the `distribute-cpus-across-cores` option for the CPUManager static policy. This feature is currently in alpha and hidden by default, marking a strategic shift aimed at optimizing CPU utilization and improving system performance across multi-core processors.
+
+## Understanding the Feature
+
+
+Traditionally, Kubernetes' CPUManager tends to allocate CPUs as compactly as possible, typically packing them onto the fewest number of physical cores. However, allocation strategy matters, CPUs on the same physical host still share some resources of the physical core, such as the cache and execution units, etc.
+
+{{< figure src="cpu-cache-architecture.png" caption="cpu-cache-architecture" >}}
+
+While default approach minimizes inter-core communication and can be beneficial under certain scenarios, it also poses a challenge. CPUs sharing a physical core can lead to resource contention, which in turn may cause performance bottlenecks, particularly noticeable in CPU-intensive applications.
+
+The new `distribute-cpus-across-cores` feature addresses this issue by modifying the allocation strategy. When enabled, this policy option instructs the CPUManager to spread out the CPUs (hardware threads) across as many physical cores as possible. This distribution is designed to minimize contention among CPUs sharing the same physical core, potentially enhancing the performance of applications by providing them dedicated core resources.
+
+
+Technically, within this static policy, the free CPU list is reordered in the manner depicted in the diagram, aiming to allocate CPUs from separate physical cores.
+
+{{< figure src="cpu-ordering.png" caption="cpu-ordering" >}}
+
+
+## Enabling the Feature
```

Comment:
Should be sentence case per [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#headings)
```suggestion
## Enabling the feature
```